edx_dl/parsing.py

```diff
@@ -268,7 +268,7 @@ def extract_courses_from_html(self, page, BASE_URL):
                     course_state = 'Started'
                 # The id of a course in edX is composed by the path
                 # {organization}/{course_number}/{course_run}
-                course_id = course_soup.a['href'][9:-5]
+                course_id = course_soup.a['href'][9:-8]  # modified by kyilmaz80
             except KeyError:
```

Comment:
I have two requests here:

1. it's not needed to annotate your changes with comments with your name. The repository already has everything attributed to you.
2. Instead of such comment, can you put a comment (better yet, a longer commit message) explaining the change in length? In fact, having these magic constants is a bad idea, even if I may have included one of those in the past.

Reply:
@kyilmaz80, First of all, sorry for the late reply.

Regarding the fixed positions/magic numbers, yes, perhaps we should split on known delimiters (or, perhaps, as one ancient version of my code used to do, use a regular expression so that the user would supply an URL and we would "magically" detect the parts of interest).

The regular expression part can certainly wait, though and I believe that the use of splitting on `/` is the way to go.